modules/page_base.py

```diff
@@ -347,6 +347,18 @@ def double_click(self, name: str, labels=[]):
         EC.element_to_be_clickable(elem)
         self.actions.double_click(elem).perform()
 
+    def triple_click_element(self, elem: WebElement, labels=[]):
```

Comment:
I'm going to do something to this method. I think we can combine these two without too many problems.